vectordb_bench/backend/clients/chroma/config.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import SecretStr
+from ..api import DBConfig
+
+class ChromaConfig(DBConfig):
+    name: str = ""
```

Comment:
I don't notice any usage of the param `name`, please remove it if it's not essential for connecting to the server.